regions/shapes/ellipse.py

```diff
@@ -258,27 +262,32 @@ def as_mpl_selector(self, ax, active=True, sync=True, callback=None,
         ``selector.set_active(True)`` or ``selector.set_active(False)``.
         """
         from matplotlib.widgets import EllipseSelector
+        from .utils import MPL_VERSION
 
         if hasattr(self, '_mpl_selector'):
-            raise Exception('Cannot attach more than one selector to a '
-                            'region.')
+            raise Exception('Cannot attach more than one selector to a region.')
```

Comment:
should not be a bare exception... I realize that isn't added in this PR, but this should be perhaps `AttributeError`?

Reply:
Good catch, thanks! I'm already updating some of the other exceptions in #390, but this is not related to rotations, and will probably be merged here faster. `AttributeError` sounds reasonable to me.